crates/interpreter/src/valid.rs

```diff
@@ -453,6 +456,21 @@ impl SideTable {
         })
     }
 
+    fn pop_cnt(source: SideTableBranch, target: SideTableBranch) -> MResult<u32, Check> {
+        let source = source.stack;
+        let target = target.stack;
+        let Some(delta) = source.checked_sub(target) else {
+            #[cfg(feature = "debug")]
+            eprintln!("side-table negative stack delta {source} < {target}");
```

Comment:
I know I suggested the current version, but when I debugged last time, I think it's clearer to just output the operation, than the unexpected true property.
```suggestion
            eprintln!("side-table negative stack delta {source} - {target}");
```